packages/grid/src/types/index.ts

```diff
@@ -158,3 +159,88 @@ export interface IRowProps extends HTMLAttributes<HTMLDivElement> {
   /** Applies the `flex-wrap` container property to the row on extra-large screens */
   wrapXl?: Wrap;
 }
+
+export const ARRAY_ORIENTATION = ['top', 'bottom', 'start', 'end'] as const;
+export type ORIENTATION = typeof ARRAY_ORIENTATION[number];
+export type DIMENSIONS = 'rows' | 'columns';
+export type UNITS = 'px' | 'fr';
+
+export interface IPaneProviderReturnProps {
+  /** Gets layout value by dimension and key. Accepts optional units.
+   *
+   * @param {DIMENSIONS} "row" or "column" dimension
+   * @param {string} Sets the splitter key to get a layout value for
+   * @param {UNITS} [units] Specifies the dimension unit for the return value
+   *
+   */
+  getLayoutValue: (dimension: DIMENSIONS, key: string, units?: UNITS) => number;
```

Comment:
It could also be that the consumer is best served by `getColumnValue` and `getRowValue` functions that take `(key, isPixel?)` parameters. Let's really push on this and make sure the API feels really nice for the end user.